README.md

```diff
@@ -1 +1,32 @@
-# mealMaster
\ No newline at end of file
+# MealMaster
+
+Web app to help with your nutritional needs.
+
+## Description
+
+The MealMaster web app aims to meet the nutritinal needs of someone living a healthy balanced lifestyle. 
+MealMaster allows the user to calculate and identify their nutritional goals through an interactive quiz.
+With the aid of a Calendar, the user can track the goals for today and prepare for the days and weeks ahead.
+MealMaster allows the user to record nutritional information, giving feedback of the remaining nutrition values for the day. 
+Information is stored to the users browser storage, allowing the user to easily monitor progress on return.
+
+## Screenshot
+
+tba
+
+## Installation
+n/a
+
+## Usage
+
+The Web app can used directly from your browser. <a href"" >link</a>
```

Comment:
Can we change this to -

The MealMaster web app can be used directly from your browser through the <a href""> link </a>